dispatch/dispatch.h

```diff
@@ -49,7 +49,7 @@
 
 #if defined(__linux__) && defined(__has_feature)
 #if __has_feature(modules)
-#include <stdio.h> // for off_t (to match Glibc.modulemap)
+//#include <stdio.h> // for off_t (to match Glibc.modulemap)
```

Comment:
a pull request is not the right discussion forum IMO, you should move that discussion to the corelibs-dev mailing list.